package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "postinstall": "bower install",
     "deploy": "npm run lint && npm run test && scripts/deploy.sh",
-    "watch": "gulp watch",
+    "watch": "gulp build && gulp watch",
```

Comment:
Correct—for any script "name", the npm script "prename" will run before, and the script "postname" will run after. This is the mechanism by which Bower will auto-install after npm install, due to the "postinstall" hook